blog/views.py

```diff
@@ -0,0 +1,102 @@
+from django.shortcuts import render
+from blog.models import Articles ,Tags, Article_Tag_Mapping, Comments
+from django.contrib import messages
+from user.models import Users
+from django.http import HttpResponse, HttpResponseRedirect
+import random
+import string
+
+# Create your views here.
+def create(request):
+    if request.method == "POST":
+        form = (request.POST)
+        check = Users.objects.get(user_id=request.user.id)
+        slug = get_slug()
```

Comment:
The good practice would be ovveride the save method of the Articles model and then create slug field there..
refer: https://fazle.me/auto-generating-unique-slug-in-django/